includes/compatibility/class-themes-compat.php

```diff
@@ -271,6 +300,20 @@ public function theme_enfold_tax_title( $args, $id ) {
 		return $args;
 	}
 
+	/**
+	 * Display the BD-plugin title if Enfold header is disabled.
+	 *
+	 * @since x.x
+	 */
+	public function theme_enfold_maybe_add_single_title() {
+		/** @phpstan-ignore-next-line */
```

Comment:
Can we remove this ignore?

Reply:
I got a GitHub check error and find some codes act like this!

Reply:
I'm not seeing what the error was, but I'm guessing it was undefined function? If so, we need to add avia_header_setting() into the stubs.php in order to remove this ignore statement.